config/example-config.json

```diff
@@ -1,8 +1,8 @@
 {
   "apiDomain": "http://localhost:3000/api",
+  "apiBasePath": "/api/auth",
+  "websiteBasePath": "/auth",
   "websiteDomain": "http://localhost:3000",
-  "apiBasePath": undefined,
-  "websiteBasePath": undefined,
   "wsBaseURL": "http://localhost:5000",
   "showTestNetworks": false,
   "grpcBCHNodeURL": "bchd.greyh.at:8335",
```

Comment:
Let's remove the actual example URL here.

Reply:
only for BCH or for both? Maybe change to something like `bchd.example.url`?

Reply:
How about just localhost:8335?

Reply:
I think is a little misleading because most people wouldn't run their own GRPC servers but use a third party; whereas the other places where `localhost` appears are places where using it as `localhost` requires no additional set up of a third party service (such as a BCH GRPC node)